packages/vscode-extension/package.json

```diff
@@ -931,7 +931,7 @@
       },
       {
         "command": "fx-extension.teamsAgentTroubleshootError",
-        "title": "Troubleshoot error with @teamsapp",
+        "title": "%teamstoolkit.commmands.teamsAgentResolve.title%",
```

Comment:
Resolve error with @teamsapp
To be consistent, let's change troubleshoot to resolve.